server/db/recreate-schema.sql

```diff
@@ -34,5 +34,6 @@ CREATE TABLE documents (
   submission_date  DATE NOT NULL,
   due_date         DATE NOT NULL,
   owner_id         uuid REFERENCES users(id) NOT NULL,
-  format           format_types
+  format           format_types,
+  content          text
```

Comment:
I think we can define the content field a non nullable as well.

Reply:
Done